src/main/webapp/js/form.js

```diff
@@ -0,0 +1,70 @@
+(function ($) {

```

Comment:
We are still confused as to where this is called and what exactly this does.
Please comment this file.

Reply:
added a more clarifying comment